README.md

```diff
@@ -1,65 +1 @@
-oVirt DPDK Setup
```

Comment:
You can fix it, but don't kill the README.md. It is good to have one.

Reply:
In the commit message you should say that you actually *rewrite*, not just refactor. And you can say how (create a module to be executed on each host, etc)

Reply:
Done

Reply:
Can you post this PR with an added commit for me to see the changes? A non-me reviewer would need better docs anyway.